test/functional/test_framework/test_node.py

```diff
@@ -919,6 +919,8 @@ def send_cli(self, clicommand=None, *args, **kwargs):
             # Ignore cli_stdout, raise with cli_stderr
             raise subprocess.CalledProcessError(returncode, p_args, output=cli_stderr)
         try:
+            if not cli_stdout.strip():
+                return None
             return json.loads(cli_stdout, parse_float=decimal.Decimal)
         except (json.JSONDecodeError, decimal.InvalidOperation):
             return cli_stdout.rstrip("\n")
```

Comment:
nit: checking for `cli_stdout` shouldn't be outside the `try`? also cos it shouldn't fail and if it does the `except` is also accessing it...
```suggestion
            raise subprocess.CalledProcessError(returncode, p_args, output=cli_stderr)
        if not cli_stdout.strip():
            return None
        try:
            return json.loads(cli_stdout, parse_float=decimal.Decimal)
        except (json.JSONDecodeError, decimal.InvalidOperation):
            return cli_stdout.rstrip("\n")
```

Reply:
I think this was more to keep the logic in one place